lib/components/BlameLine.js

```diff
@@ -21,12 +21,14 @@ export default function BlameLine(props) {
   } = props;
 
   const displayName = showOnlyLastNames ? lastWord(author) : author;
-
+  const showHash = atom.config.get('git-blame.showHash');
```

Comment:
I'd like to keep this react component component stateless and not rely on outside state store besides whats passed in via `props`. Can you move this config check to the render side of `BlameLine` component and pass it in as a `prop`?

Then we can just use the `showHash` prop itself below.